packages/theme-chalk/src/common/var.scss

```diff
@@ -695,7 +695,7 @@ $button: map.merge(
     'active-border-color': getCssVar('color-primary'),
     'active-bg-color': getCssVar('button', 'hover-bg-color'),
     'outline-color': getCssVar('color-primary', 'light-5'),
-    'hover-link-text-color': getCssVar('color-info'),
+    'hover-link-text-color': #909399,
```

Comment:
I am not sure exactly what the problem is. Can you explain?

The problem seems to be that variables cannot be redefined. But why do you delete the existing variable and put it as a constant?

Reply:
According to the discussion of #16979, in the default theme, when `el-button` is set to link but not type, the hover state text color uses the text color of type info, which is normal.

However, if the theme color is changed, if the color corresponding to info is set to blue when the type of `el-button` is not specified, the hover state text color becomes blue, which does not seem reasonable.